code/ARAX/test/test_ARAX_expand.py

```diff
@@ -475,9 +475,17 @@ def test_molepro_query():
 @pytest.mark.external
 def test_spoke_query():
     actions_list = [
-        "add_qnode(ids=OMIM:603903, categories=biolink:PhenotypicFeature, key=n00)",
-        "add_qnode(categories=biolink:PhenotypicFeature, key=n01)",
-        "add_qedge(subject=n00, object=n01, key=e00)",
+        "add_qnode(ids=NCBIGene:7157, categories=biolink:Gene, key=n00)", "add_qnode(categories=biolink:ChemicalEntity, key=n01)", "add_qedge(subject=n01, object=n00, key=e00, predicates=biolink:molecularly_interacts_with)",
```

Comment:
one minor style thing: probably a good idea to keep each "add_qnode.." or "add_qedge" on its own line, just for readability

Reply:
Oh yeah I thought I did that but must have not saved it in my IDE.